src/utils/storeUniqueToken.ts

```diff
@@ -0,0 +1,39 @@
+import { env } from "../typeDefinitions/default.types";
+import jwt from "@tsndr/cloudflare-worker-jwt";
+
+export const storeUniqueToken = async (
+  token: string,
+  discordId: number,
+  env: env
+) => {
+  const jwtToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 15 },
+    env.PRIVATE_TOKEN,
+    { algorithm: "RS256" }
+  );
+  const data = {
+    type: "discord",
+    token: token,
+    attributes: {
```

Comment:
why do we need this `attribute` key?

Reply:
It is mentioned in the Data Model by @rohan09-raj 
![image](https://user-images.githubusercontent.com/86982322/221248321-86102a10-2cb2-4634-95e5-48f23033fc5c.png)